app/assets/javascripts/reducers/stories.js

```diff
@@ -20,6 +20,11 @@ const storiesReducer = (state = initialState, action) => {
         updateIfSameId(action.story.id, (story) => {
           return updateStory(story, action.story);
         }));
+    case actionTypes.DELETE_STORY_SUCCESS:
+    return state.map(
```

Comment:
what about a `state.filter` here?

also, I would suggest you to move this logic to a function instead of writing it inside the reducer's  `switch`